ts/features/itwallet/common/components/discoveryBanner/ItwDiscoveryBanner.tsx

```diff
@@ -67,11 +71,24 @@ export const ItwDiscoveryBanner = ({
   };
 
   const bannerConfig = {
-    onboarding: {
-      content: I18n.t("features.itWallet.discovery.banner.home.content"),
-      title: I18n.t("features.itWallet.discovery.banner.home.title"),
-      action: I18n.t("features.itWallet.discovery.banner.home.action")
-    },
+    // TODO: Once offline access is fully integrated, the "homeWithOffline" copy can be merged into "home" and this dynamic logic can be removed.
+    onboarding: isOfflineAccessEnabled
```

Comment:
Could we use `useItwDiscoveryBannerType` instead? Adding a new banner type might be a cleaner solution.